packages/local/src/index.ts

```diff
@@ -0,0 +1,150 @@
+import { Context, Logger, Random, Schema } from 'koishi'
+import { ImageSource } from 'koishi-plugin-booru'
+import { pathToFileURL } from 'node:url'
+import { readFile, writeFile } from 'node:fs/promises'
+import { existsSync } from 'node:fs'
+import { resolve } from 'node:path'
```

Comment:
use old-school import for node built-in modules, e.g: `node:url` -> `url`.
Because we should still support node 14 which didn't support this.